src/renderer/components/ft-list-video/ft-list-video.scss

```
@@ -3,3 +3,8 @@
 .thumbnailLink:hover {
   outline: 3px solid var(--side-nav-hover-color);
 }
+
+.thumbnailImage.thumbnailLoading {
+  // Makes img element sized correctly before image loading starts
+  aspect-ratio: 16 / 9;
```

According to the [aspect-ratio docs on MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/aspect-ratio#fallback_to_natural_aspect_ratio), you can do this for images, which will make it use 16/9 until the image is downloaded and after that use the image's aspect ratio.

That would allow you to remove the javascript and the thumbnail loading class.
```suggestion
  aspect-ratio: 16/9 auto;
```

---

And it's done